README.md

```diff
@@ -4,28 +4,25 @@
 
     $ yarn install stylelint-config-primarysite stylelint
 
-With your project's `.stylelintrc` file, extend 'stylelint-config-primarysite',
-like so:
+With your project's `.stylelintrc` file, extend 'stylelint-config-primarysite', like so:
 
     {
        "extends": "stylelint-config-primarysite"
     }
 
-*Note*: This package expects that `postcss-cssnext` is being used. If it's not,
-the following rules should be used in your config to override this package:
-
-    'at-rule-blacklist': 'import',
-    'selector-pseudo-class-blacklist': null
+_Note_: Installing this package's optional `peerDependencies` will change the behaviour of some of
```

Comment:
Do I need to give more detail here?

Reply:
It would be helpful to extend this comment otherwise it's very vague. You don't have to add much more though, I think something like `..change the behaviour of some rules to account for the additonal features provided by the optional dependenies` is enough.